reference/docs-conceptual/getting-started/fundamental/Getting-Detailed-Help-Information.md

```diff
@@ -1,170 +1,203 @@
+- [about_Functions](/powershell/module/microsoft.powershell.core/about/about_functions)
+- [about_Scripts](/powershell/module/microsoft.powershell.core/about/about_scripts)
+- [about_Comment_Based_Help](/powershell/module/microsoft.powershell.core/about/about_comment_based_help)
 
-To get Help online, try the *Online* parameter of the Get-Help cmdlet. The *Online* parameter of the Get-Help cmdlet works only for cmdlet Help, function Help, and script Help. You cannot use the *Online* parameter with conceptual (About) topics or provider Help topics. Also, because this feature is optional, it does not work for every cmdlet, function, or script Help topic.
+## Getting help online
 
-However, all the Help topics that come with Windows PowerShell, including provider Help and conceptual (About) Help topics, are available online in the [Windows PowerShell](http://go.microsoft.com/fwlink/?LinkID=107116) section of the Microsoft TechNet Library.
+Viewing the Help articles online is one of the best ways to get help. Online articles are easier
+to update and provide the most current content.
 
-To use the *Online* parameter of the Get-Help cmdlet, use the following command format.
+To get Help online, use the **Online** parameter of the `Get-Help` cmdlet. All the Help articles
+that come with PowerShell, including provider Help and conceptual (About) Help articles, are
+available online in the [PowerShell](/powershell/scripting/powershell-scripting) documentation.
 
-```
-get-help <command-name> -online
-```
+> [!NOTE]
+> You can't use the **Online** parameter with conceptual (about_*) or provider Help articles.
+> Online help is optional, so it does not work for every cmdlet, function, or script.
 
-For example, to get the online version of the Help topic about the Get-ChildItem cmdlet, type:
+For example, to get the online version of the Help article about the `Get-ChildItem` cmdlet, type:
 
+```powershell
+Get-Help Get-ChildItem -Online
 ```
-get-help get-childitem -online
-```
-
-If an online version of the Help topic is available, it will open in your default browser.
 
-If online Help is supported for a Help topic, you can also view the Internet address (URL) of the Help topic. The Internet address appears in the Related Links section of a Help topic.
+PowerShell opens the article in your default browser. If online Help is supported for a Help
+article, you can also view the URL of the Help article. The URL appears in the Related Links
+section of a Help article.
 
 For example, to see the URL for the online version of the Add-Computer cmdlet, type:
 
-```
-get-help add-computer
+```powershell
+Get-Help Add-Computer
 ```
 
-The first line in the Related Links section of the topic is shown below.
+The first line in the Related Links section of the article is shown below.
 
+```powershell
```

Comment:
Is this really PowerShell or it it just output or text in a file (metadata)?

Reply:
This is a line of output from `Get-Help`